OPERATOR.rst

```diff
@@ -156,28 +156,45 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
+Operators must check for updates to the GitLab instances and the GitLab Runners
+on a daily basis. Updates to the Gitlab instance can be quickly checked at the
```

Comment:
```suggestion
on a daily basis. Updates to Gitlab can be quickly checked at the
```

Using the word "instance" here is confusing. Primarily, we're looking for updates to the webapp and runner images, not the instance. We have to update the instance if either them are updated.